continuous_integration/environment-3.8-dev.yaml

```diff
@@ -7,7 +7,7 @@ dependencies:
   - python=3.8.*
   # For the dependencies. These will be re-installed later.
   - numpy
-  - pandas
+  - pandas=1.0.*
```

Comment:
pinned here to older pandas, since otherwise I get the failures that are being handled by https://github.com/dask/dask/pull/6996